android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/home/HomeScreen.kt

```diff
@@ -59,11 +86,39 @@ private fun HomeScreenContent(
 @Preview
 @Composable
 fun HomeScreenContentLoadingPreview() {
-    HomeScreenContent("TUESDAY, NOVEMBER 8", "image_url", true)
+    HomeScreenContent(
+        HomeUiModel(
+            "TUESDAY, NOVEMBER 8",
+            "image_url",
+            HomeSurveysUiModel(
+                "Working from home Check-In",
+                "We would like to know what are your goals and skills you wanted",
+                "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l",
+                3,
+                1,
+                true
+            ),
+            true
+        )
+    )
 }
 
 @Preview
 @Composable
 fun HomeScreenContentPreview() {
-    HomeScreenContent("TUESDAY, NOVEMBER 8", "image_url", false)
+    HomeScreenContent(
+        HomeUiModel(
+            "TUESDAY, NOVEMBER 8",
+            "image_url",
+            HomeSurveysUiModel(
+                "Working from home Check-In",
+                "We would like to know what are your goals and skills you wanted",
+                "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l",
+                3,
+                1,
+                false
+            ),
+            false
+        )
```

Comment:
It would be better if we apply [Multipreview Annotations](https://developer.android.com/jetpack/compose/tooling#preview-multipreview) to get rid of duplicated Preview for different states. Could you please give it a try?

Reply:
Updated in https://github.com/suho/kmm-ic/pull/106/commits/292662b6eebc6d1b2d93c12abc3ad4e6f6ecc4b7